pilot/pkg/serviceregistry/kube/controller/endpoint_builder.go

```diff
@@ -131,7 +131,7 @@ func (b *EndpointBuilder) buildIstioEndpoint(
 		ServiceAccount:        b.serviceAccount,
 		Locality:              b.locality,
 		TLSMode:               b.tlsMode,
-		Address:               endpointAddress,
+		Addresses:             []string{endpointAddress},
```

Comment:
Does this mean we are never setting 2 addresses...?

Reply:
Hi @howardjohn, I changed here in advance, please see the new submit.

Reply:
In case of dualstack, the secondary IP is appended to the list at L279